src/Tizen.NUI/src/public/Common/StyleManager.cs

```diff
@@ -37,6 +37,17 @@ public class StyleManager : BaseHandle
         private EventHandler<StyleChangedEventArgs> styleManagerStyleChangedEventHandler;
         private StyleChangedCallbackDelegate styleManagerStyleChangedCallbackDelegate;
 
+        /// <summary>
+        /// The Type of BrokenImage
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public enum BrokenImageType
+        {
+            SMALL = 0,
```

Comment:
We usually use PascalCasing.(Samll, Normal, Large)
https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions